tests/modules/programs/eww/basic-configuration.service

```
@@ -0,0 +1,13 @@
+[Install]
+WantedBy=graphical-session.target
+
+[Service]
+ExecReload=@eww@ reload
+ExecStart=@eww@ daemon --no-daemonize
+ExecStop=@eww@ kill
```

```suggestion
ExecReload=@eww@/bin/eww reload
ExecStart=@eww@/bin/eww daemon --no-daemonize
ExecStop=@eww@/bin/eww kill
```
`@eww@` is a stub for the Nix store path for Eww if I understand correctly.

---

That I didn't know, thank you for having a look at it. I learned something new too :)

---

Same.